.cirrus.yml

```diff
@@ -272,8 +272,6 @@ task:
     - name: hostonly_devicelab_tests-3_last-linux
       << : *LINUX_SHARD_TEMPLATE
 
-    # TODO(ianh): name: add_to_app_tests-linux
```

Comment:
Intentional?

Reply:
Oh, I get it now, these would correspond to the one you're removing for other platforms. LGTM.

Reply:
There's no more add_to_app_tests on any platform, so we don't need a windows or linux one.